docs/challenges/guides/style.md

```diff
@@ -0,0 +1,45 @@
+# Challenge Style
+
+Always presents the original DataFrame in the context/assignment of
```

Comment:
Typo: Always presents --> Always present

Reply:
fixed in c6c37a6d545